Automattic/wp-calypso
# Add WP Admin link to Global Site Sidebar

Related to 
- close https://github.com/Automattic/dotcom-forge/issues/5628
- https://github.com/Automattic/wp-calypso/pull/87576

## Proposed Changes

This PR adds the WP Admin link to the Global Site Sidebar.
Design: pfsHM7-aY-p2

<img width="282" alt="Screenshot 2024-02-19 at 15 58 26" src="https://github.com/Automattic/wp-calypso/assets/5287479/6dc1e8f4-748a-485e-8914-4b2d8b49c763">


## Testing Instructions



- Open Calypso Live
- Go to any Calypso page (e.g., /home)
- See how the sidebar looks like

## Pre-merge Checklist
